HW05 Jerry Lee.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "HW05 Jerry Lee"
+author: "jerry lee"
+date: "8/7/2020"
+output: html_document
+---
+
+# HW05
+Due August 3rd
+
+### HW Assignment
+
+This week the goal is to practice tidying data and joining files together. Similar to last week you can practice these skills with data you already have or you can follow with the two (probably pretty short) HW assignments below.
+
+## 1. Tidying the dad_mom file
+
+In this repo, you'll find a dad_mom.txt file that isn't particularly tidy. Clean this up so there are only 4 columns 
+Tidy this data frame so that it adheres to the tidy data principles:
+
+    Each variable must have its own column.
+    Each observation must have its own row.
+    Each value must have its own cell.
+
+```{r loading packages}
+library(tidyverse)
+library(ggplot2)
+library(forcats)
+```
+
+```{r reading data #1}
+dad_mom<-read_tsv("dad_mom.txt")
+
+dad_mom
+```
+
+```{r tidying data}
+dad_mom_tidy<-dad_mom%>%
+  unite("dad",name_dad:income_dad,sep="_")%>% #lines 43-44 to associate incomes with each person
+  unite("mom",name_mom:income_mom,sep="_")%>%
+  gather(key="parent_type",value="name",dad,mom)%>% #turn each row into an observation
+  separate(name,c("name","income"),sep="_") #create separate columns for name and income variables
+
```

Comment:
I agree that you do a great job of detailing your work with your commented code. And the method that you used to tidy the data looks clean and efficient.